packages/autocomplete/src/index.ts

```diff
@@ -49,6 +50,10 @@ export const MATCH_COMPLETIONS = ([] as AnyCompletions).concat(
   BSON_TYPES
 );
 
+const DB_AGGREGATE_COMPLETIONS = STAGE_OPERATORS.filter(({ namespaces }) => {
+  return namespaces.length === 1 && (namespaces as readonly string[]).includes(DATABASE);
```

Comment:
You’re already checking that `namespaces` only contains one value here, so it should be possible to avoid the cast here without too much trouble:

```suggestion
  return namespaces.length === 1 && namespaces.[0] === DATABASE;
```

It might also be worth adding a comment about this not being all `db.aggregate` completions as the name might imply, but only first-stage completions?

Reply:
Oh that's right. Thanks for pointing that out.

Reply:
About the comment regarding `DB_AGGREGATE_COMPLETIONS` variable itself. I made the usage of it more explicit which conveys that we are only providing operators that are first stage only.